bank-account/bank_account_tests.erl

```diff
@@ -0,0 +1,95 @@
+-module(bank_account_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+
+create_test() ->
+	      Bank_account = bank_account:create(),
+	      ?assert(bank_account:balance( Bank_account ) =:= 0).
+
+close_account_test() ->
+              Bank_account = bank_account:create(),
+              bank_account:deposit( Bank_account, 1 ),
+              Amount = bank_account:close( Bank_account ),
+              ?assert(Amount =:= 1),
+	      ?assertError(function_clause, bank_account:balance( Bank_account )).
```

Comment:
Again, it'd be great to line up the statements in this file. Probably another tabs/spaces issue.